### PR TITLE
Photoshop: create single frame image in Ftrack as review

### PR DESCRIPTION
**Pull request by kalisp**
## Brief description
Previously PS was creating .mov even for single frame review. This PR allows to create and upload single image instead.

## Description
Next paragraf is more elaborate text with more info. This will be displayed for example in collapsed form under the first sentence in a changelog.

## Additional info
Still in progress as Play button under Versions metadata is finicky, Play button on thumbnail in version list seems to work fine.

## Testing notes:
Screenshot 2022-09-20 165806

1. Double check jpg options for Photoshop ExtractReview in Setting
2. Publish workfile that has single or no `image` instance.
3. Check that `publish/review` folder contains high res review
4. Check in Ftrack by clicking on Thumbnail in Versions list that high res image pops up

### CODE REVIEW

File: openpype/modules/ftrack/plugins/publish/integrate_ftrack_instances.py [hunk 1]
```diff
@@ -147,6 +151,8 @@ def process(self, instance):
         first_thumbnail_component = None
         first_thumbnail_component_repre = None
         for repre in thumbnail_representations:
+            if review_representations and not has_movie_review:
+                break
```
Comment (BigRoy) [inline openpype/modules/ftrack/plugins/publish/integrate_ftrack_instances.py:155]: Both these variables don't seem to be related to the `repre` we are currently iterating and thus it feels off that we're checking these variables inside the for loop instead of prior to it? (It looks confusing because it reads as if related to the current repre being iterated)

**Comment by kalisp** [comment]
There is still discrepancy between clicking on thumbnail in Version list and Side panel 'Play' https://discord.com/channels/@me/928645495822368799/1026857661368651856 which is being solved with Ftrack Support.

Customer is fine going ahead with this for now, so we can go ahead and test it

**Review by 64qam — APPROVED** [review]
